srv/modules/runners/mgr_orch.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# vim: ts=8 et sw=4 sts=4
+# pylint: disable=too-few-public-methods,modernize-parse-error
+"""
+Runner endpoints for integration specifically with the ceph-mgr orchestrator
+"""
+
+from __future__ import absolute_import
+import logging
+import os
+import sys
+# pylint: disable=import-error,3rd-party-module-not-gated,redefined-builtin
+import salt.client
+
+log = logging.getLogger(__name__)
+
+
+def _sanitize_devices(devices):
+    """
+    Takes a list of devices and returns only the information necessary for
+    the ceph-mgr orchestrator's InventoryDevice class.
+    """
+
+    # Shamelessly stolen from populate.py, to try to prefer /dev/disk/by-id
```

Comment:
If this is a 1-1 copy from another module we should think about making it public and importable

Reply:
I agree in principle, but rather than making this function, and the one from srv/modules/runners/populate.py, a public importable thing, I'd rather figure out how to take the one from srv/salt/_modules/cephdisks.py:device_()  and use that in all _three_ places. Except this might all change with ceph-volume, so I'd kinda like to leave it for later :-)